Scripts/Core/EditorVR.Tools.cs

```diff
@@ -190,23 +189,39 @@ static bool SelectTool(Transform rayOrigin, Type toolType)
 					if (deviceData.rayOrigin == rayOrigin)
 					{
 						var spawnTool = true;
-
-						// If this tool was on the current device already, then simply remove it
-						if (deviceData.currentTool != null && deviceData.currentTool.GetType() == toolType)
+						var currentTool = deviceData.currentTool;
+						var currentToolType = currentTool.GetType();
+						var currentToolIsSelect = currentToolType == typeof(SelectionTool);
+						var setSelectAsCurrentTool = toolType == typeof(SelectionTool) && !currentToolIsSelect;
```

Comment:
Can we have some other way of marking a tool as important/default for EVR rather than hardcoding the selectiontool aspect?

Reply:
@AndrewTHEManeri  We definitely could/should go about doing this.  Though, would this be something you'd want in this PR?  If so, we could discuss arch.  If not, we could do it in a follow-up PR.

Reply:
It can be a separate PR but I would want it to be happening really soon

Reply:
Now tracked via #282